UnitsNet/Scripts/UnitDefinitions/Length.json

```diff
@@ -70,6 +70,18 @@
       ]
     },
     {
+      "SingularName": "USSurveyFoot",
+      "PluralName": "USSurveyFeet",
+      "FromUnitToBaseFunc": "x*0.3048006096",
+      "FromBaseToUnitFunc": "x/0.3048006096",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "ftUS" ]
```

Comment:
This link says `ft US`: http://georeference.org/doc/units.htm

What do you think, both or one of them?